backend/coins/btc/db/transactionsdb/transactionsdb_test.go

```diff
@@ -146,9 +149,12 @@ func TestTxQuick(t *testing.T) {
 				require.NoError(t, tx.MarkTxVerified(txHash, expectedHeaderTimestamp))
 				delete(allUnverifiedTxHashes, txHash)
 				require.True(t, checkTxHashes())
-				_, _, _, headerTimestamp, err := tx.TxInfo(txHash)
+				_, _, _, headerTimestamp, createdTimestamp, err := tx.TxInfo(txHash)
 				require.NoError(t, err)
 				require.Equal(t, expectedHeaderTimestamp.String(), headerTimestamp.String())
+				now := time.Now()
+				require.NotNil(t, createdTimestamp)
```

Comment:
For existing app installs, this will be `nil`, I guess. What happens then, does everything work?

Reply:
Yeah, this PR has all instances of accessing this field. They all check for nil.

Reply:
Ah sorry! I somehow missed byHeight's sort.Interface.